internal/dao/stubs/context_getters.go

```diff
@@ -0,0 +1,42 @@
+package stubs
+
+import (
+	"context"
+
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+)
+
+type daoStubCtxKeyType int
+
+const (
+	accountCtxKey daoStubCtxKeyType = iota
+	pubkeyCtxKey  daoStubCtxKeyType = iota
+)
+
+func WithPubkeyDao(parent context.Context, init_store []*models.Pubkey) context.Context {
```

Comment:
I think it is supposed to be `initStore` because go supports camelCase

Reply:
Agreed and also commented by @lzap, but I'm moving code here, so I'd like to move as is and address this as a whole thing and completely change the implementation of Pubkey stub in a follow-up